src/components/Layout/Header.tsx

```diff
@@ -3,12 +3,12 @@ import React from 'react';
 const Header = () => {
   return (
     <div className="et-header">
-      <div className="et-header__center">
+      <div className="et-header__center" id="et-header">
         <div className="et-logo et-hidden-xs">
           KeepCent <span>(1.0.0)</span>
         </div>
-        &nbsp;
-        <input className="et-header-search-bar" placeholder="Search for a transaction..." />
+        {/* &nbsp;
```

Comment:
We had a search box here now you commented it. If we don't need this feature please remove the code.

Reply:
Thanks for your feedback, I'll remove this one for now until we make a decision about having or not having search-box.